src/components/TrackersManager.vue

```diff
@@ -0,0 +1,309 @@
+<script setup lang="ts">
+import { type ZodAffectType } from '@/types/zodAffect';
+import { useTrackers } from '@/composables/useTrackers';
+import { computed, toRef } from 'vue';
+
+const props = defineProps<{
+  flawId: string;
+  affects: ZodAffectType[];
+  mode: 'modal' | 'embedded';
+}>();
+
+const affects = toRef(props, 'affects');
```

Comment:
This is only used on line 26, on line 29 and 40 you use `props.affects`
I think you can remove it and also use `props.affects` on line 26